TypeCobol/Compiler/SqlCodeElements/SqlStatementType.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TypeCobol.Compiler.CodeElements;
+
+namespace TypeCobol.Compiler.SqlCodeElements
+{
+    /// <summary>
+    /// List all types of Sql executable statements
+    /// </summary>
+    public enum SqlStatementType
```

Comment:
I don't think C# enums are made to be extended. I fear that we have to stick with `CodeElementType` and `StatementType`. Although it forces us to mix sql with the rest, it is clearer and safer for future readers/commiters in the project.

Reply:
removed in 5a26682